### PR TITLE
Add styles to signup and signin views

### PR DESCRIPTION
**Pull request by adrianMRJ**
# Description
- What?: We've added styles to signup and login views
 - Why?: These changes were necesary for the visuals of the project.

  ## Screenshots
Sign-up
Sign-up (1)
Sign-in
Sign-in (1)

### CODE REVIEW

File: stylesheets/application/objects/buttons.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  margin-top: 6rem;
+  margin-bottom: 3rem;
+}
+
+
+@media screen and (min-width: $breakpoint-tablet) {
+  .formbutton{
+    margin-top: 3rem;
+  }
+}
```
Comment (JuanequeX) [inline stylesheets/application/objects/buttons.scss:20]: Add blank line here

File: stylesheets/application/objects/inputs.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.forminput {
+    background-color: #EBEAEA;
+    border-radius: .2rem;
+    border: solid .05rem #D9D9D9;
+    padding: .3rem;
+}
```
Comment (JuanequeX) [inline stylesheets/application/objects/inputs.scss:6]: Add blank line here

File: stylesheets/application/objects/labels.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    font-weight: 700;
+}
+
+.nodecoration{
+    text-decoration: none;
+}
+
+.signuptag{
+    font-weight: bold;
+}
```
Comment (JuanequeX) [inline stylesheets/application/objects/labels.scss:21]: Add blank line here

Comment (JuanequeX) [reply stylesheets/application/objects/labels.scss:21]: And modify this identation

File: stylesheets/application/typography.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+body{
+  font-family: 'Roboto', sans-serif;
+}
```
Comment (JuanequeX) [inline stylesheets/application/typography.scss:5]: Add blank line here

File: stylesheets/application/objects/inputs.scss (hunk 2 above)
Comment (JuanequeX) [reply stylesheets/application/objects/inputs.scss:6]: And modify this identation

File: stylesheets/application/objects/buttons.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+.formbutton {
+  border: none;
+  background-color: #464343;
+  color: white;
+  border-radius: .2rem;
+  height: 2rem;
+  width: 100%;
+  font-weight: bold;
+  margin-top: 6rem;
+  margin-bottom: 3rem;
```
Comment (JuanequeX) [inline stylesheets/application/objects/buttons.scss:12]: Please sort this block

File: stylesheets/application.scss [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+      font-size: 1.1rem;
+      background-color: #444444;
+    }
+
+    .container{
+      background-color: #F6F6F6;
+      border-radius: 1.5rem;
+      width: fit-content;
+      padding-top: 2rem;
+    }
```
Comment (JuanequeX) [inline stylesheets/application.scss:43]: solve this identation

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: stylesheets/application/objects/buttons.scss (hunk 1 above)
Comment (adrianMRJ) [reply stylesheets/application/objects/buttons.scss:20]: done

File: stylesheets/application/objects/inputs.scss (hunk 2 above)
Comment (adrianMRJ) [reply stylesheets/application/objects/inputs.scss:6]: Done

File: stylesheets/application/objects/labels.scss (hunk 3 above)
Comment (adrianMRJ) [reply stylesheets/application/objects/labels.scss:21]: Done

File: stylesheets/application/typography.scss (hunk 4 above)
Comment (adrianMRJ) [reply stylesheets/application/typography.scss:5]: Done

File: stylesheets/application/objects/buttons.scss (hunk 5 above)
Comment (adrianMRJ) [reply stylesheets/application/objects/buttons.scss:12]: Done

File: stylesheets/application.scss (hunk 6 above)
Comment (adrianMRJ) [reply stylesheets/application.scss:43]: done

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by jestradadeveloper — APPROVED** [review]
LGTM

**Pull request merged**